database/snapshotOptions/templateExport.json

```diff
@@ -12,9 +12,10 @@
     "templateSection",
     "templateStageReviewLevel",
     "permissionName",
-    "permissionPoliy"
+    "permissionPoliy",
```

Comment:
Spelling: `permissionPolicy"

Reply:
Also -- are you sure we want to export "permissionPolicy"? (It's currently not being, cos it's spelled wrong, and it seems fine).

Reply:
removed:
89241a246aafa5cf9f22d83a35f1cc64a29cc45f